string sentence
---
string msg_body
string msg_id
string outcome
